src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/comparator/LatLong_AbsoluteDifferenceSim_Comparator.java

```diff
@@ -0,0 +1,54 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.Comparator;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.ComparatorLogger;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.Matchable;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.similarity.numeric.AbsoluteDifferenceSimilarity;
+
+public class LatLong_AbsoluteDifferenceSim_Comparator implements Comparator<Car, Attribute> {
+
+        private static final long serialVersionUID = 1L;
+
+        private AbsoluteDifferenceSimilarity sim = new AbsoluteDifferenceSimilarity(75); // TODO what to use here?
+        private ComparatorLogger comparisonLog;
+
+        @Override
+        public double compare(Car record1, Car record2, Correspondence<Attribute, Matchable> schemaCorrespondence) {
+
+            Double lat1 = record1.getRegion().getLatitude();
```

Comment:
NullPointer.

Reply:
I have the check in my branch - strange..
if( lat1 != null && lat2 != null && long1 != null && long2 != null) {...